config/site/src/_includes/filtering_predicate_definitions/conjunctions.md

```diff
@@ -6,11 +6,11 @@
   filters that can't be provided on a single filter input because of collisions between key names.
   For example, if you want to provide multiple `anySatisfy: ...` filters, you could do `allOf: [{anySatisfy: ...}, {anySatisfy: ...}]`.
 
-  Will be ignored when `null` or an empty list is passed.
+  Will be treated as `true` when `null`, an empty object or list is passed.
 
 [`anyOf`]({% link query-api/filtering/conjunctions.md %})
 : Matches records where any of the provided sub-filters evaluate to true.
   This works just like an `OR` operator in SQL.
 
-  Will be ignored when `null` is passed. When an empty list is passed, will
-  cause this part of the filter to match no documents.
+  Will be treated as `true` when `null` or an empty object is passed.
+  When an empty list is passed, will cause this part of the filter to match no documents.
```

Comment:
"Will be treated as `true`" and "cause this part of the filter to match no documents" are opposites but aren't phrased as such, so it's an odd pairing.  Maybe we should standardize on one of these pairs:

* "Will be treated as `true`"/"Will be treated as `false`"
* "cause this part of the filter to match all documents"/"cause this part of the filter to match no documents"

Overall, I'm thinking that "matches all documents"/"matches no documents" is clearer than "treated as `true`"/"treated as `false`".  In the context of a SQL WHERE clause, where you can use a literal `true` or `false`, "treated as (true|false)" makes a lot of sense, but we don't let users provide a literal true or false in our GraphQL filter syntax. So maybe it's worth rewording all the docs to speak in terms of "matching all documents" and "matching no documents" rather than true/false.

That's a loosely held opinion, though, and I realize it essentially involves redoing this PR.  But if there's consensus that it's clearer, probably worth it.

@BrianSigafoos-SQ I'm curious what you think?

Reply:
I would prefer to keep it "Will be treated as `true`"/"Will be treated as `false`", since we end up using that phrase in other parts of the project (e.g. tests). Though I can be swayed either ways depending on the consensus.

Reply:
That SGTM. I'd also consider making it more consistent and slightly less wordy.

From this:
```
Will be treated as `true` when `null` is passed.

When an empty list is passed, will cause this part of the filter to match no documents.
```

To this: 

```
When `null` is passed, matches all documents.

When an empty list is passed, this part of the filter matches no documents.
```